odoo/addons/base/migrations/13.0.1.3/pre-migration.py

```diff
@@ -567,6 +568,7 @@
     "sale_order_line_invoice_rel",
     "summary_dept_rel",
     "project_task_assign_so_line_rel",
+    "mail_mass_mailing_tag_rel",
```

Comment:
Please put this on utm, not here.

Reply:
why?

Reply:
Otherwise, I should move every table in that dictionary to each module.

Reply:
Well, it's not the same that the model dissappears than this is a functional change belonging to that module. Anyway, I would prefer if it's the case to have all the FK dropping in its respective module if proceeds.

Reply:
But the model in this case also disappears. There is no functional change. The model mail.mass.mailing.campaign is not renamed and thus becomes obsolete, the table mail_mass_mailing_campaign becomes obsolete and the mail_mass_mailing_tag_rel the same. Don't see the difference.

Reply:
> to have all the FK dropping in its respective module if proceeds.

In this PR or in another?

Reply:
But if the model disappears, and we don't change anything, why do we need to drop FK constraints?

Reply:
Quoting @Yajo in two cases:

A) "to avoid unknown cascade behaviors to slow down or break the DB structure."
B) "in a DB that was migrated from v10 and that had many procurement orders takes days. That's because the FK was set without an index."

Reply:
OK, I remember this now, thanks. Maybe a code comment can serve for refreshing this. Let's keep it here for not moving more.